gammapy/data/data_store.py

```diff
@@ -242,32 +248,42 @@ def get_observations(self, obs_id=None, skip_missing=False, required_irf="all"):
             Observation IDs (default of ``None`` means "all")
         skip_missing : bool, optional
             Skip missing observations, default: False
-        required_irf : list of str
+        required_hdu : list of str
             Runs will be added to the list of observations only if the
-            required IRFs are present. Otherwise, the given run will be skipped
+            required HDUs are present. Otherwise, the given run will be skipped
             Available options are:
+            * `events` : Events
+            * `gti` :  Good time intervals
             * `aeff` : Effective area
             * `bkg` : Background
             * `edisp`: Energy dispersion
             * `psf` : Point Spread Function
-            By default, all the IRFs are required.
+            * `rad_max` : Maximal radius
+            * `full-containment` :["events", "gti", "aeff", "edisp", "psf", "bkg"]
+            * `point-source` : ["events", "gti", "aeff", "edisp", "rad_max"]
```

Comment:
`rad_max` is not required for `point-like` (it should be `point-like`, not `point-source`!) since the `RAD_MAX` header keyword can also be used.

See this issue: #3717  and corresponding PR: #3731

Reply:
thanks modified